format/spec.md

```diff
@@ -443,13 +443,13 @@ The schema of a manifest file is a struct called `manifest_entry` with the follo
 | _optional_ | _optional_ | **`132  split_offsets`**          | `list<133: long>`            | Split offsets for the data file. For example, all row group offsets in a Parquet file. Must be sorted ascending |
 |            | _optional_ | **`135  equality_ids`**           | `list<136: int>`             | Field ids used to determine row equality in equality delete files. Required when `content=2` and should be null otherwise. Fields with ids listed in this column must be present in the delete file |
 | _optional_ | _optional_ | **`140  sort_order_id`**          | `int`                        | ID representing sort order for this file [3]. |
-
+| _optional_ | _optional_ | **`141  spec_id`**                | `int`                        | ID representing partition spec for this file [4]. |
```

Comment:
Well, if we inherit from the manifest I think we may not want to add this here since it may confuse folks since this spec_id doesn't matter even if it's written. Maybe this change should just be limited towards clarifying why 141 is reserved. Thoughts @Fokko @rdblue

Reply:
It is definitely confusing. I think should not reserve an ID for it if we don't write it.

Reply:
This isn't optional. The field ID is reserved, but it should never be written to a metadata file.

We need to reserve an ID for it because it is present in metadata tables. We don't want to have conflicting IDs.

Reply:
+1 on not adding it here but mentioning this ID is reserved.

Reply:
How about keeping it blank? This means that the field should not be written.
```suggestion
| | | **`141  spec_id`**                | `int`                        | ID representing partition spec for this file [4]. |
```

Reply:
I think it's a bit confusing to have a type that isn't included in v1 or v2. I'd remove it from here and maybe clarify what it is in the footnote.